_opensearch/install/rpm.md

```diff
@@ -5,154 +5,472 @@ parent: Install OpenSearch
 
-```bash
-sudo systemctl start opensearch-performance-analyzer.service
-```
-
-To stop the Performance Analyzer, enter:
+You should add trust for the root certificate to your client before sending requests. If you do not add trust, then you must use the `-k` option so that cURL ignores CN and root certificate validation.
+{:.tip}
 
 ```bash
-sudo systemctl stop opensearch-performance-analyzer.service
+$ curl https://your.host.address:9200 -u admin:yournewpassword -k
+{
+  "name" : "hostname-here",
+  "cluster_name" : "opensearch",
+  "cluster_uuid" : "efC0ANNMQlGQ5TbhNflVPg",
+  "version" : {
+    "distribution" : "opensearch",
+    "number" : "2.1.0",
+    "build_type" : "tar",
+    "build_hash" : "388c80ad94529b1d9aad0a735c4740dce2932a32",
+    "build_date" : "2022-06-30T21:31:04.823801692Z",
+    "build_snapshot" : false,
+    "lucene_version" : "9.2.0",
+    "minimum_wire_compatibility_version" : "7.10.0",
+    "minimum_index_compatibility_version" : "7.0.0"
+  },
+  "tagline" : "The OpenSearch Project: https://opensearch.org/"
+}
 ```
 
-## Upgrade RPM
+## Upgrade to a newer version
 
-You can upgrade your RPM OpenSearch instance both manually and through YUM. 
+OpenSearch instances installed using RPM or YUM can be easily upgraded to a newer version. We recommend updating with YUM, but you can also upgrade using YUM.
 
 
-### Manual 
+### Manual upgrade with rpm 
 
-Download the new version of OpenSearch you want to use, and then use `rpm -Uvh` to upgrade.
+Download the RPM package for the desired upgrade version directly from the [OpenSearch downloads page](https://opensearch.org/downloads.html){:target='\_blank'}.
+
+Navigate to the directory containing the distribution and run the following command:
+```bash
+rpm -Uvh opensearch-{{site.opensearch_version}}-linux-x64.rpm
+```
 
 ### YUM
 
-To upgrade to the latest version of OpenSearch with YUM, use `sudo yum update`. You can also upgrade to a specific OpenSearch version by using `sudo yum update opensearch-<version-number>`.
+To upgrade to the latest version of OpenSearch with YUM:
+```bash
+sudo yum update
+```
+
+ You can also upgrade to a specific OpenSearch version:
+ ```bash
+ sudo yum update opensearch-<version-number>
+ ```
+
+## Related links
+
+- [OpenSearch configuration]({{site.url}}{{site.baseurl}}/opensearch/configuration/)
+- [Configure Performance Analyzer for Tarball Installation]({{site.url}}{{site.baseurl}}/monitoring-plugins/pa/index/#configure-performance-analyzer-for-tarball-installation)
+- [Install and configure OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/dashboards/install/index/)
+- [OpenSearch plugin installation]({{site.url}}{{site.baseurl}}/opensearch/install/plugins/)
+- [About the security plugin]({{site.url}}{{site.baseurl}}/security-plugin/index/)
```

Comment:
maybe also add link to RPM YUM section here:
[YUM method](({{site.url}}{{site.baseurl}}/opensearch/install/rpm/#yum-method)